Vagrantfile

```diff
@@ -1,9 +1,10 @@
-
 Vagrant.configure("2") do |config|
   config.vm.box = "ubuntu/xenial64"
```

Comment:
Do consider using a newer Ubuntu version. You might find python3 already installed and in the requested version